pkg/cliplugins/workload/plugin/syncer.yaml

```diff
@@ -42,6 +42,14 @@ rules:
   - "get"
   - "watch"
   - "list"
+- apiGroups:
+  - "networking.k8s.io"
+  resources:
+  - networkpolicies
+  verbs:
+  - "create"
+  - "list"
+  - "watch"
```

Comment:
no delete, no get, no update?

Reply:
not in the PR. I'm trying to struck a balance between small reviewable PRs vs big ones. So no delete/update in this PR.

Reply:
related issue: https://github.com/kcp-dev/kcp/issues/2358